.devcontainer/postStartCommand.sh

```diff
@@ -2,13 +2,21 @@
 #git remote add upstream https://github.com/twjudson/aata
 #git remote add origin https://github.com/jbrock1999h/aata
 
+# The codespace tends to start under the main branch by default, and if the remotes aren't set up correctly you cannot change it to master. This is the line of code that would allow you to automatically set the branch to master if you have origin set (if you know the URL for the user's repo):
+#git checkout --track origin/master
```

Comment:
See https://docs.github.com/en/codespaces/developing-in-codespaces/default-environment-variables-for-your-codespace -- I think that `GITHUB_REPOSITORY` has what you need to set origin correctly.